scripts/integration.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Get record key from env
+export $(cat .env | grep CYPRESS_RECORD_KEY | xargs)
+RECORD_KEY="${CYPRESS_RECORD_KEY}"
+
+export PRIVATE_KEY=0x0054b824c2083e7db09f36edb2ab24eb31f8276fa6cd62e30b42e3a185b37179
```

Comment:
id put a quick comment here about where this private key is from

Reply:
This is ganache account. Extracted from dev.sh 
https://github.com/DXgovernance/dxvote/blob/b0cca80c4b9fc3a521958523e6e6bcc46749023d/scripts/dev.sh#L25